addon/components/learnergroup-selection-manager.hbs

```diff
@@ -1,27 +1,12 @@
 <div class="learnergroup-selection-manager" data-test-learnergroup-selection-manager>
-  <div class="selected-learner-groups" data-test-selected-learner-groups>
-    <h4 data-test-title>
-      {{t "general.selectedLearnerGroups"}}
-    </h4>
-    {{#if @learnerGroups.length}}
-      <DetailLearnergroupsList
-        @learnerGroups={{@learnerGroups}}
-        @isManaging={{true}}
-        @remove={{@remove}}
-      />
-    {{else}}
-      <p data-test-no-selected-learnergroups>
-        {{t "general.none"}}
-      </p>
-    {{/if}}
-  </div>
+  <SelectedLearnerGroups @learnerGroups={{await @learnerGroups}} @isManaging={{true}} @remove={{@remove}}/>
```

Comment:
This is an example of why I'm leaning towards removing these awaits. The value getting passed in should probably be resolved before it is used here otherwise we risk passing promises all over the place and it's hard to see what they do.

Reply:
:heavy_check_mark: